packages/perseus/src/widgets/interactive-graphs/reducer/use-graph-state.ts

```diff
@@ -0,0 +1,18 @@
+import React, {createContext} from "react";
+
+import type {InteractiveGraphAction} from "./interactive-graph-action";
+import type {InteractiveGraphState} from "../types";
+
+export const GraphStateContext = createContext<{
+    state: InteractiveGraphState;
+    dispatch: React.Dispatch<InteractiveGraphAction>;
+}>({
+    state: {} as any,
+    dispatch: () => {},
+});
+
+export default function useGraphState() {
+    const {state, dispatch} = React.useContext(GraphStateContext);
+
+    return {state, dispatch};
```

Comment:
Why do we destructure `state` and `dispatch` and then bundle them up into a new object to return them? Could we do this instead?

```suggestion
export default function useGraphState() {
    return React.useContext(GraphStateContext);
```